### PR TITLE
cheap etherium chain

### PR DESCRIPTION
**Pull request by McCrearyD**
https://cheapeth.org

the coin is forked from etherium and everything is the same except 1000cTH = 1ETH

### CODE REVIEW

File: _data/chains/eip155-777.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "symbol": "cTH",
+    "decimals": 18
+  },
+  "infoURL": "https://cheapeth.org",
+  "shortName": "cTH",
+  "chainId": 777,
+  "networkId": 777,
+  "slip44": 60,
+  "ens": {
+    "registry":"0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"
```
Comment (ligi) [inline _data/chains/eip155-777.json:20]: are you sure about this?

Comment (McCrearyD) [reply _data/chains/eip155-777.json:20]: lol, honestly I didn't mean to PR it I wanted to see the CI test runs

Comment (ligi) [reply _data/chains/eip155-777.json:20]: so i can close this?

Comment (McCrearyD) [reply _data/chains/eip155-777.json:20]: yes, apologies!

**Pull request closed without merging**